src/pages/ProductSell/ProductSell.jsx

```diff
@@ -1,7 +1,299 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import SellSubmit from './component/SellSubmit';
 
 const ProductSell = () => {
-  return <div>ProductSell</div>;
+  const [photo, setPhoto] = useState([]);
+  const [imgList, setImgList] = useState([]);
+  const [sellList, setSellList] = useState({
+    title: '',
+    subCategoryId: 0,
+    region: '',
+    conditionId: 0,
+    exchangeable: '',
+    price: 0,
+    description: '',
+    userId: 1,
+  });
```

Comment:
```suggestion
  const [sellList, setSellList] = useState({
    title: '',
    subCategoryId: 0,
    region: '',
    conditionId: '1',
    exchangeable: '1',
    price: 0,
    description: '',
    userId: 1,
  });
```

다혜님 그 라디오 버튼 구현하는 부분
제 맥북으로는 잘 돼서 한번 적용해 보시고 적용 된다면 이렇게 수정해주세요!